src/printer.js

```diff
@@ -628,15 +628,17 @@ function genericPrintNoParens(path, options, print) {
           concat([
             leftBrace,
             indent(
-              options.tabWidth * (parentIsUnionTypeAnnotation ? 2 : 1),
+              options.tabWidth + (parentIsUnionTypeAnnotation ? 3 : 0),
```

Comment:
@yamafaktory That's a good point, yep I think we should do what @rhengles said (sorry for the slow reply)